package.json

```diff
@@ -44,7 +44,8 @@
 	},
 	"homepage": "https://github.com/ProphetFund/peanut-sdk#readme",
 	"dependencies": {
-		"ethersv5": "npm:ethers@^5"
+		"ethersv5": "npm:ethers@^5",
+		"viem": "^1.19.15"
```

Comment:
Why are we adding viem dependency? We should have either ethers or viem, but not both. Keep in mind that integrators need to add these dependencies to their codebase too.

I know viem is quite lightweight (and paradigm has been makign rlly good tech), so I'm in favor of considering a full on switch to viem, but it's not in scope rn

Reply:
I really like viem's utils like `sliceHex` 😭  (and I haven't seen any of these utils in ethers)

So, it was a try, let me remove it now :) 😭😭😭